public/docs/ts/latest/guide/user-input.jade

```diff
@@ -37,27 +40,27 @@ include ../_util-fns
 .l-main-section
 :marked
   ## Get user input from the $event object
-  We can bind to all kinds of events. Let's bind to the keyup event of an input box and replay
+  You can bind to all kinds of events. In this case, you'll bind to the keyup event of an input box and replay
```

Comment:
This page has a lot of "will/'ll" in it, which is counter to our style... I think our editor gave up on doing the kind of rewrite it'd take to eradicate it, but you should feel free to go further (as long as doing so doesn't make the text unfriendly or hard to read).